client/rest-high-level/src/test/java/org/elasticsearch/client/ml/EvaluateDataFrameResponseTests.java

```diff
@@ -33,37 +33,29 @@
-        List<EvaluationMetric.Result> metrics;
-        switch (evaluationName) {
-            case OutlierDetection.NAME:
-                metrics = randomSubsetOf(
-                    Arrays.asList(
-                        AucRocResultTests.randomResult(),
-                        PrecisionMetricResultTests.randomResult(),
-                        RecallMetricResultTests.randomResult(),
-                        ConfusionMatrixMetricResultTests.randomResult()
-                    )
-                );
-                break;
-            case Regression.NAME:
-                metrics = randomSubsetOf(
-                    Arrays.asList(MeanSquaredErrorMetricResultTests.randomResult(), RSquaredMetricResultTests.randomResult())
-                );
-                break;
-            case Classification.NAME:
-                metrics = randomSubsetOf(
-                    Arrays.asList(
-                        AucRocResultTests.randomResult(),
-                        AccuracyMetricResultTests.randomResult(),
-                        org.elasticsearch.client.ml.dataframe.evaluation.classification.PrecisionMetricResultTests.randomResult(),
-                        org.elasticsearch.client.ml.dataframe.evaluation.classification.RecallMetricResultTests.randomResult(),
-                        MulticlassConfusionMatrixMetricResultTests.randomResult()
-                    )
-                );
-                break;
-            default:
-                throw new AssertionError("Please add missing \"case\" variant to the \"switch\" statement");
-        }
+        List<EvaluationMetric.Result> metrics = switch (evaluationName) {
+            case OutlierDetection.NAME -> randomSubsetOf(
+                Arrays.asList(
+                    AucRocResultTests.randomResult(),
+                    PrecisionMetricResultTests.randomResult(),
+                    RecallMetricResultTests.randomResult(),
+                    ConfusionMatrixMetricResultTests.randomResult()
+                )
+            );
+            case Regression.NAME -> randomSubsetOf(
+                Arrays.asList(MeanSquaredErrorMetricResultTests.randomResult(), RSquaredMetricResultTests.randomResult())
+            );
+            case Classification.NAME -> randomSubsetOf(
+                Arrays.asList(
+                    AucRocResultTests.randomResult(),
+                    AccuracyMetricResultTests.randomResult(),
+                    org.elasticsearch.client.ml.dataframe.evaluation.classification.PrecisionMetricResultTests.randomResult(),
+                    org.elasticsearch.client.ml.dataframe.evaluation.classification.RecallMetricResultTests.randomResult(),
+                    MulticlassConfusionMatrixMetricResultTests.randomResult()
+                )
+            );
+            default -> throw new AssertionError("Please add missing \"case\" variant to the \"switch\" statement");
```

Comment:
According to [JEP 354](https://openjdk.java.net/jeps/354#Exhaustiveness), we shouldn't add `default` cases like these anymore. Switch expressions are exhaustive, and so the compiler ought to tell us if the enum and this switch don't match.

Reply:
Argh, ignore me - I thought this was switching over an enum. I'll go check my other suggestions.